loefsys/users/urls.py

```diff
@@ -1,8 +1,8 @@
 from django.urls import path, include
 
-from members.views import ProfileDetailView, ProfileListView
+from users.views import ProfileDetailView, ProfileListView
 
-app_name = "members"
+app_name = "users"
```

Comment:
we should discuss this naming, and maybe even move this logic to the base module of the system. As it not only covers users, but also external contacts.

Reply:
Moved users to the base module of the system. Have yet to come up with a better name for it though.